ci/install.sh

```diff
@@ -10,6 +10,9 @@ install_c_toolchain() {
             sudo apt-get install -y --no-install-recommends \
                  gcc-aarch64-linux-gnu libc6-arm64-cross libc6-dev-arm64-cross
             ;;
+	*-musl)
+	    sudo apt-get install musl-tools
```

Comment:
Hmm, no. I'm certain that install.sh runs after the apt-get packages get installed. Instead it seems that no apt-get packages are installed for the stable i686-musl target; I don't know why though. There's no "Installing APT Packages (BETA)" at the top. See

[x86_64](https://travis-ci.org/japaric/rust-everywhere/jobs/163442310#L237) vs
[i686](https://travis-ci.org/japaric/rust-everywhere/jobs/163442305#L237)